pkgs/top-level/all-packages.nix

```diff
@@ -25557,4 +25557,5 @@ in
 
   gortr = callPackage ../servers/gortr {};
 
+  hyperspace = callPackage ../applications/misc/hyperspace { };
```

Comment:
Please don't add it at the end but under a fitting headline and alphabetically sorted.